ci-operator/config/stackrox/stackrox/stackrox-stackrox-remove_python_from_central-db.yaml

```diff
@@ -656,6 +656,6 @@ tests:
     test:
     - ref: stackrox-initial
 zz_generated_metadata:
-  branch: master
```

Comment:
As I understand, this is needed only to test https://github.com/stackrox/stackrox/pull/3880.

Would it be possible to keep the configuration for the `master` branch and just add the configuration for the `remove_python_from_central` branch? So that we have both in parallel. I assume that `master` branch builds will be broken with this change until we revert this PR?

Also, it may be good to add support for the new branch and leave everything else as it is (`nightlies`, `3.73`, etc.). And then, when we see that CI works and the stackrox branch can be merged, we can prepare a new PR on CI that will fix `nightlies`, `3.73`, and also remove support for the new branch. After that, we can merge both PRs in a short time to avoid possible broken CI.

Reply:
So I tried that before. To keep master config and this branch but then I have problems with image "promotions" and finally for some reason images were not build for branch rehearsals.
Anyway having both configurations has no benefit for us. After merging to master CI on branch should pass as its targeting master.